piccolo/apps/asgi/commands/new.py

```diff
@@ -92,9 +93,9 @@ def new(root: str = ".", name: str = "piccolo_project"):
 
             if file_name.endswith(".jinja"):
                 output_file_name = file_name.replace(".jinja", "")
-                template = Environment(
-                    loader=FileSystemLoader(searchpath=dir_path)
-                ).get_template(file_name)
+                template = Environment(loader=FileSystemLoader(searchpath=dir_path)).get_template(
```

Comment:
@tarsil Linter complains because the line is too long (here and in few other places).  Piccolo uses `79` as the line length. [Here](https://piccolo-orm.readthedocs.io/en/latest/piccolo/contributing/index.html#code-style) are the code style docs.
First you need to run linters and formaters locally with this set of commands:
1. Run formaters: `./scripts/format.sh`
2. Run linters: `./scripts/lint.sh`

That should fix the linting error. Hope that helps.

Reply:
@sinisaos perfect. Forgot the `format.sh`. Done it now.

Appreciated your help on this and now the CI should work fine.